graduates.html

```diff
@@ -41,6 +41,237 @@ <h1 class="header-h1 grid-item">Refcode</h1>
       <h2>Refcode Graduates</h2>
       <p>Students who successfully completed all the course requirements are listed below. Professional referrals or references for any of the students listed below are available upon request.</p>
 
+      <h3>2019 Cohort</h3>
+
+      <figure class="graduates-picture2">
+        <img src="media/graduates&volunteers2019.jpg">
+        <figcaption>Graduates and volunteers from the 2019 coding class.</figcaption>
+        <style scoped>
+        .graduates-picture2 img {
+          clip-path: inset(20px 0px 20px 0px);
+          margin-top: -20px;
+          margin-bottom: -20px;
+        }
+        </style>
+      </figure>
+      <h3>Graduates</h3>
+      <div class="student">
+        <h4 class="student-name">Abel</h4>
+        <div class="student-photo">
+          <img src="media/Abel.png"/>
+        </div>
+        <ul class="student-links">
+          <li>Website: <a href="http://djabelmixmaster.com/" rel="external">djabelmixmaster.com</a></li>
+          <li><a href="https://github.com/djabelbas" rel="external">GitHub Profile</a></li>
+          <li><a href="#" rel="external">LinkedIn Profile</a></li>
+        </ul>
+      </div>
+      <div class="student">
+        <h4 class="student-name">Bezawit</h4>
+        <div class="student-photo">
+          <img src="media/Bezawit.png"/>
+        </div>
+        <ul class="student-links">
+          <li>Website: <a href="http://bezu.ga/" rel="external">bezu.ga</a></li>
+          <li><a href="https://github.com/bez" rel="external">GitHub Profile</a></li>
+          <li><a href="#" rel="external">LinkedIn Profile</a></li>
+        </ul>
+      </div>
+
+      <div class="student">
+        <h4 class="student-name">Bizuayehu</h4>
+        <div class="student-photo">
+          <img src="media/Bizuayehu.png"/>
+        </div>
+        <ul class="student-links">
+          <li>Website: <a href="http://bizuayehu.ga/" rel="external">bizuayehu.ga</a></li>
+          <li><a href="https://github.com/bizuayehu1968/Soli" rel="external">GitHub Profile</a></li>
```

Comment:
This should link to the profile not the repo, so: https://github.com/bizuayehu1968